metaseq/checkpoint_utils.py

```diff
@@ -543,7 +543,7 @@ def load_model_ensemble_and_task(
         assert num_shards > 0
         for shard_idx in range(num_shards):
             if num_shards == 1:
-                filename = filename.replace(".pt", suffix + ".pt")
+                filename = re.sub('.pt$', suffix + ".pt", filename)
```

Comment:
@sriniiyer  did you test if this still works for the old models?

Reply:
Should we upstream this? How did we end up here?

Reply:
@tbmihailov  I tried this with 1.3B_gptz_azure and it works

@stephenroller We added a new flag to the checkpoint filename called pt_XX, so its mainly a problem for us. Upstream doesnt have this problem.